<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1061</width>
    <height>744</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SlimFeed</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QTableView" name="feedList"/>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="childrenCollapsible">
        <bool>false</bool>
       </property>
       <widget class="QTableView" name="entryList"/>
       <widget class="QWebView" name="browser">
        <property name="url">
         <url>
          <string>about:blank</string>
         </url>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1061</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFeed">
    <property name="title">
     <string>&amp;Feed</string>
    </property>
    <addaction name="actionAdd"/>
    <addaction name="actionMarkAllFeedsRead"/>
    <addaction name="actionFetchAllFeeds"/>
   </widget>
   <widget class="QMenu" name="menuArticle">
    <property name="title">
     <string>&amp;Article</string>
    </property>
    <addaction name="actionMarkAllEntriesRead"/>
   </widget>
   <widget class="QMenu" name="menuSlimfeed">
    <property name="title">
     <string>&amp;Slimfeed</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
    <addaction name="actionPreferences"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuSlimfeed"/>
   <addaction name="menuFeed"/>
   <addaction name="menuArticle"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAdd">
   <property name="text">
    <string>&amp;Add</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionMarkAllFeedsRead">
   <property name="text">
    <string>Mark All As &amp;Read</string>
   </property>
  </action>
  <action name="actionMarkAllEntriesRead">
   <property name="text">
    <string>Mark All As &amp;Read</string>
   </property>
  </action>
  <action name="actionFetchAllFeeds">
   <property name="text">
    <string>&amp;Fetch All Feeds</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About SlimFeed</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>&amp;Preferences...</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Q&amp;t</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
